src/libs/AxelarQueryAPI.ts

```diff
@@ -308,4 +314,73 @@ export class AxelarQueryAPI {
       });
     }
   };
+
+  public async getTransferLimit({
+    chainId,
+    denom,
+  }: {
+    denom: string;
+    chainId: string;
+  }): Promise<string> {
+    try {
+      // verify chain params
+      await throwIfInvalidChainIds([chainId], this.environment);
+      const chains = await loadChains({ environment: this.environment });
+      const chain = chains.find((c) => c.id === chainId);
+      if (!chain) throw `Chain ${chainId} not found`;
+
+      // verify asset params
+      if (!this.allAssets) await this._initializeAssets();
+      const asset = this.allAssets.find((asset) => asset.common_key[this.environment] === denom);
+      if (!asset) throw `Asset ${denom} not found`;
+
+      // if chain is on evm module, query the contract directly on the EVM chain
+      // othewise if the chain is in the axelarnet module, query the nexus module
+      return formatUnits(
```

Comment:
I would prefer the return value to be a subunit integer rather than a unit with fractions (e.g. `123 cents` not `1.23 dollars`) because it is likely that devs will use this function to compare with another value and it's much easier to do math operation in BigNumber with an integer than a float. The reason is `BigNumber` really hates fractional numbers. 
<img width="911" alt="image" src="https://user-images.githubusercontent.com/78221556/210742742-38828e96-8a1c-4441-875d-b90d3fcd1611.png">

I think the `formatUnits` function should only be used before we render it to the users to make it looks more "user-friendly".

So I think we can just do
```ts
return chain.module === "evm"
          ? this.queryEVMTransferLimit(chainId, asset, chain)
          : this.queryAxelarTransferLimit(chainId, denom)
```